omnibus/config/projects/chef.rb

```diff
@@ -20,7 +20,7 @@
 homepage "https://www.chef.io"
 
 build_iteration 1
-build_version "12.6.0"
+build_version "12.7.0"
```

Comment:
Feedback requested:

So we don't have to keep updating this in lockstep with the `version.rb` file, does anyone care if use a `File.open` to read the current version and load it here?  The other option would be to update the `rake version` task to also update this version string.

Since we already use `rake version` to update the chef-config version file I think that makes the most sense.  Feedback?